projects/canopy/src/lib/shared/breakpoints.interface.ts

```diff
@@ -1 +1,9 @@
 export type Breakpoints = 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
+
+export enum BreakpointValues {
```

Comment:
It might be one for the future but I learnt a bit from this article..
https://blog.logrocket.com/why-you-should-use-css-env-9ee719ce0f24/

We may be able to reduce the replication of these variables across css and js.

Reply:
I just noticed the spacing variables replication too. I wouldn't be too opposed to these being replicated, but it might be nice to log an issue and reference it in the code for both the css and js, just so people know they need to update both.

Reply:
Thanks Andy, I'd like propose that we add an issue and do this in another PR, perhaps on one of a dedicated Canopy day, IMO getting this feature in sooner outweighs the benefits of having this duplication and waiting to do the css env implementation. Happy to discuss though